src/coding-guidelines/expressions.rst

```diff
@@ -82,6 +82,59 @@ Expressions
 
          fn with_base(_: &Base) { ... }
 
+.. guideline:: Do not use integer type as divisor
+   :id: gui_7y0GAMmtMhch
+   :category: advisory
+   :status: draft
+   :release: latest
+   :fls: fls_Q9dhNiICGIfr
+   :decidability: decidable
+   :scope: module
+   :tags: numerics
```

Comment:
I think we want to add here the tag `subset` and perhaps `reduce-human-error` as well